packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <div
+        :class="$style['c-mediaElement']"
+        data-test-id="mediaElement-component">
+        mediaElement
+    </div>
+</template>
+
+<script>
+
+export default {
+    name: 'MediaElement',
+    components: {},
```

Comment:
I'd remove these properties if they aren't being used yet.

Reply:
Should this also be a change in the generator? Will make change